examples/zap/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	zapbrake "github.com/airbrake/gobrake/v5/zap"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var ProjectID int64 = 999999                               // Insert your Project ID here
+var ProjectKey string = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" // Insert your Project Key here
+
+var notifier = gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+	ProjectId:   ProjectID,
+	ProjectKey:  ProjectKey,
+	Environment: "production",
+})
+
+/*
+Note: This example only shows how to send errors to Airbrake
+i.e. how to create a core and send the logs to airbrake
+If you want to write the logs to json file or stdout
+you should use zapcore.NewTee() for using multiple cores
+You can set the severity accordingly
```

Comment:
```suggestion
Note: This example only shows how to send errors to Airbrake,
i.e., how to create a core and send the logs to Airbrake.
If you want to write the logs to a JSON file or stdout,
you should use `zapcore.NewTee()` for using multiple cores.
You can set the severity accordingly.
```

Reply:
Do we need commas before and after `i.e`?

Reply:
https://jakubmarian.com/comma-after-i-e-and-e-g/